src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static final Random random = new Random();
+    public static final Color[] values = Color.values();
+
+    public String getRandomColor() {
+        int index = random.nextInt(values.length);
+        return Color.values()[index].name().toLowerCase();
```

Comment:
There is no need to call `Color.values()` again since you have already stored it in the `values` array. Use the `values` array directly to get the color.